package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "widget-sso-saml2",
+  "version": "1.0.0",
+  "description": "Documentation is available at https://developers.fliplet.com/API/integrations/sso-saml2.html",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Fliplet/fliplet-widget-sso-saml2.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Fliplet/fliplet-widget-sso-saml2/issues"
+  },
+  "homepage": "https://github.com/Fliplet/fliplet-widget-sso-saml2#readme",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-fliplet": "^0.8.0"
```

Comment:
Not latest version https://github.com/Fliplet/eslint-config-client/blob/master/package.json

Reply:
updated the  eslint-config-fliplet version ^1.0.8